cedar-policy/tests/example_use_cases_doc.rs

```diff
@@ -68,9 +68,7 @@ fn scenario_4a() {
 // note: 4b currently omitted because it requires date/timestamp functionality
 
 /// currently failing, as the validator does not support action attributes
-#[should_panic(
-    expected = "error occurred while evaluating policy `policy0`: entity `Action::\\\"view\\\"` does not exist"
-)]
+#[should_panic(expected = "Action::\\\"view\\\"` does not have the attribute `readOnly`")]
```

Comment:
Actually having the entity specifies changes the error message.

Reply:
This overlaps with my #360

Reply:
Not surprising. We'll want to make sure yours wins.

Reply:
but I think it's the same change, so we should be able to accept both in either order